### PR TITLE
Submission - Neumorphic loading Animation - Yugesh Jain

### PR DESCRIPTION
**Pull request by YugeshJainMM**
#### Details
* A neumorphic button used to create a creative loading screen with random sizes circles popping up and disappearing

https://github.com/mutualmobile/flutter_animations_example/assets/98592058/f132acb7-f331-470e-86a0-74ece3a79354

### CODE REVIEW

File: assets/images/billieeilish.png
Comment (pushpalroymm) [inline assets/images/billieeilish.png:1]: Are these images used in the animation?

File: lib/main.dart [hunk 1]
```diff
@@ -50,6 +52,8 @@ class _AnimationsCarouselState extends State<AnimationsCarousel> {
     const AnimatedContainerScreen(),
     const FiSplashScreen(),
     const DayNightAnimation(),
+    const SliderAnimation(),
```
Comment (pushpalroymm) [inline lib/main.dart:55]: Have we added 2 animations in this PR?

**Comment by pushpalroymm** [comment]
@YugeshJainMM I think the Slider animation changes are also added in this PR by mistake. Can we remove and reformat them?

**Comment by YugeshJainMM** [comment]
Done👍🏻
removed slider animation code and assets

File: lib/animations/neumorphic_loading_animation.dart [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              duration: const Duration(milliseconds: 200),
+              height: widget.initialSize.height,
+              width: widget.initialSize.width,
+              decoration: BoxDecoration(
+                color: Colors.grey[300],
+                borderRadius: BorderRadius.circular(50),
+                boxShadow: widget.isPressed
+                    ? [
+                        BoxShadow(
+                          color: Colors.grey[500]!,
```
Comment (aditya-bhawsar-mm) [inline lib/animations/neumorphic_loading_animation.dart:115]: dont use `!`

**Review by aditya-bhawsar-mm — APPROVED** [review]

File: lib/animations/neumorphic_loading_animation.dart (hunk 2 above)
Comment (YugeshJainMM) [reply lib/animations/neumorphic_loading_animation.dart]: Done!

**Review by aditya-bhawsar-mm — APPROVED** [review]

File: lib/main.dart (hunk 1 above)
Comment (YugeshJainMM) [reply lib/main.dart]: removed

**Pull request merged**